packages/ui/src/components/filters/filters-bar/actions/views.tsx

```diff
@@ -25,9 +25,14 @@ import { FilterValue, SavedView, SortValue, ViewManagement } from '../../types'
 interface ViewsProps {
   currentView: SavedView | null
   savedViews: SavedView[]
-  viewManagement: ViewManagement & {
+  viewManagement: {
     activeFilters: FilterValue[]
     activeSorts: SortValue[]
+    checkNameExists: ViewManagement['checkNameExists']
+    saveView: ViewManagement['saveView']
+    updateView: ViewManagement['updateView']
+    deleteView: ViewManagement['deleteView']
+    renameView: ViewManagement['renameView']
```

Comment:
Probably simpler to use `Pick` for this stuff
```suggestion
  viewManagement: Pick<ViewManagement, 'checkNameExists' | 'saveView' | 'updateView' | 'deleteView' | 'renameView'> & {
    activeFilters: FilterValue[]
    activeSorts: SortValue[]
```


Or in cases where you want to up everything except for a couple of things, `Omit` is handy.